nautobot_golden_config/__init__.py

```diff
@@ -28,6 +28,9 @@ class GoldenConfig(PluginConfig):
         "enable_compliance": True,
         "enable_intended": True,
         "enable_sotagg": True,
+        "enable_postprocessing": True,
```

Comment:
```suggestion
        "enable_postprocessing": False,
```
I believe we should disable this by default.

Reply:
agreed.